src/rgw/rgw_op.cc

```diff
@@ -124,7 +124,9 @@ static int parse_range(const char *range, off_t& ofs, off_t& end, bool *partial_
   return r;
 }
```

Comment:
I guess this is an improvement.  Again, this isn't something you want to do in headers, right?